independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/api/WebClientApplicationException.java

```diff
@@ -22,18 +22,41 @@
 /**
  * Subclass of {@link WebApplicationException} for use by clients, which forbids setting a response that
  * would be used by the server.
- * FIXME: I'd rather this be disjoint from WebApplicationException, so we could store the response info
- * for client usage. Perhaps we can store it in an alternate field?
+ * FIXME: I'd rather this be disjoint from WebApplicationException, but changing the exception type could break existing
```

Comment:
I considered not inheriting from WebApplicationException, as there's no use for it if I can't use it's response property... however, it may break code that is still catching it, right?
Let me know if this comment is accurate.
I'd rather have a separate exception type that has it's own response property to avoid confusion, though... right now people should know that the response is available in the new field instead of the standard one.